Makefile

```
@@ -0,0 +1,76 @@
+BUILDDIR=$(CURDIR)/output/build
+GOBIN=$(CURDIR)/output/bin
+
+GOMOBILE=$(GOBIN)/gomobile
+# Add GOBIN to $PATH so `gomobile` can find `gobind`.
+GOBIND=env PATH="$(GOBIN):$(PATH)" "$(GOMOBILE)" bind
+IMPORT_HOST=github.com
+IMPORT_PATH=$(IMPORT_HOST)/Jigsaw-Code/outline-client
```

Ugh, IMPORT_HOST just hurts readability. I see it was already there, no need to change in this PR.
```suggestion
IMPORT_PATH=github.com/Jigsaw-Code/outline-client
```